### ISSUE TITLE
Access Denied to Dev although joined to dialout

### ISSUE THREAD

**Issue by mikebones** [tribut/homeassistant-docker-venv#7]
It looks like it's starting the venv normally:
```
administrator@hme-srv-02:~/docker$ docker logs home-assistant
Not running as root. Extra packages won't be installed.
Running as 1000:1000
Setting umask to 007
Initializing venv in /var/tmp/venv
```
but i'm getting an error when attaching a zwave stick:
```
2020-05-04 18:20:23 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Z-Wave (import from configuration.yaml) for zwave
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/openzwave/option.py", line 75, in __init__
    raise ZWaveException(u"Can't write to device %s : %s" % (device, traceback.format_exception(*sys.exc_info())))
openzwave.object.ZWaveException: "Zwave Generic Exception : Can't write to device /dev/ttyACM0 : ['NoneType: None\\n']"
```
my user is in the dialout group, plus everything else seems to add up:
```
administrator@hme-srv-02:~/docker$ groups administrator
administrator : administrator adm dialout cdrom sudo dip plugdev lxd docker
administrator@hme-srv-02:~/docker$ ls -lah /dev/ttyA*
crw-rw---- 1 root dialout 166, 0 May  5 00:35 /dev/ttyACM0
administrator@hme-srv-02:~/docker$ id -u
1000
```
Any suggestions? I've tried destroying and recreating the container but no difference.

**Comment by mikebones** [comment]
I actually should have looked around first.  This is fixed with PR#6.

**Comment by tribut** [comment]
Great to hear #6 fixed this for you, so I'm closing the issue.
